packages/emf/backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/services/messages/EMFMessageService.java

```diff
@@ -39,12 +39,17 @@ public String unexpectedError() {
 
     @Override
     public String invalidInput(String expectedInputTypeName, String receivedInputTypeName) {
-        return this.messageSourceAccessor.getMessage("INVALID_INPUT", new Object[] { expectedInputTypeName, receivedInputTypeName });
+        return this.messageSourceAccessor.getMessage("INVALID_INPUT", new Object[]{expectedInputTypeName, receivedInputTypeName});
```

Comment:
The Eclipse formatting profile keeps the spaces between `[`, `{`, etc.

Reply:
Done,
I've changed my format settings